webpack.common.js

```diff
@@ -38,6 +38,14 @@ module.exports = {
           },
         ],
       },
+      {
+        test: /\.mp3$/,
```

Comment:
Move this regex to test where png,jpg etc are 
You don't need to specify the same loader twice